flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/DatabaseSync.java

```diff
@@ -128,13 +129,7 @@ public void build() throws Exception {
         }
         List<String> syncTables = new ArrayList<>();
         List<Tuple2<String, String>> dorisTables = new ArrayList<>();
-        Map<String, Integer> tableBucketsMap = null;
-        if (tableConfig.containsKey("table-buckets")) {
-            tableBucketsMap = getTableBuckets(tableConfig.get("table-buckets"));
-        }
 
-        // Set of table names that have assigned bucket numbers.
-        Set<String> tablesWithBucketsAssigned = new HashSet<>();
```

Comment:
The logic of this variable seems to be removed?

Reply:
This logic has actually been moved [here](https://github.com/apache/doris-flink-connector/pull/439/files#diff-a2601d62182c02933253309a325f29d4a2731d1e2c5e1b711a7354cd780a9083R57-R62).

@vinlee19 Please also help to review it.